deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
Why would deps break if `LLVM_VER` is an empty string? I don't think we use it for other deps and LLVM shouldn't be broken more if llvm-config isn't found for system LLVM.

This also only execute shell command if `USE_SYSTEM_LLVM` is set and in that case the `LLVM_VER` is wrong right now and IMHO it's better to break it loud rather than silently picking a wrong one.